opentelemetry/server/main.go

```diff
@@ -58,7 +58,7 @@ func main() {
 
 	p := provider.NewOpenTelemetryProvider(
 		provider.WithServiceName(serviceName),
-		provider.WithExportEndpoint("host.docker.internal:4317"),
+		provider.WithExportEndpoint(":4317"),
```

Comment:
Can it be set to `jaeger-all-in-one:4317` ? Or put it in an environment variable?